app/Http/Controllers/ApplicationController.php

```diff
@@ -146,9 +146,19 @@ public function edit($id)
       $hear_about = Scholarship::getCurrentScholarship()->hear_about_options;
       $choices = Application::formatChoices($hear_about);
 
+      // We have to pass uploads to the view, so set null if there aren't any
+      if ($application['upload'])
+      {
+        $uploads = explode(',', $application['upload']);
```

Comment:
Not necessary for merge, but this could be a great place to use an accessor/mutator to serialize this as a CSV and then deserialize to an array in the model (rather than having to remember to manually `explode` 💥).